float64 radius
---
bool result
